packages/legend-query-builder/src/components/__tests__/QueryBuilderFetchStructure.test.tsx

```diff
@@ -301,6 +301,18 @@ test(
     ).not.toBeNull();
     fireEvent.click(getByText(modal, 'Close'));
 
+    // Test text decriptions are correctly added for result set modifiers
```

Comment:
can we go to the result modifiers modal, remove all modifiers and test we no longer see the belong.

Reply:
the test is updated as requested